sdx_lc/messaging/topic_queue_consumer.py

```diff
@@ -93,22 +93,37 @@ def handle_mq_msg(self, msg_body):
         if is_json(msg_body):
             self.logger.info("JSON message")
             msg_json = json.loads(msg_body)
-            if "uni_a" in msg_json and "uni_z" in msg_json:
+            if (
+                "link" in msg_json
+                and "uni_a" in msg_json["link"]
+                and "uni_z" in msg_json["link"]
+            ):
+                connection = msg_json["link"]
                 self.logger.info("Got connection message.")
-                self.db_instance.add_key_value_pair_to_db(self.message_id, msg_body)
+                self.db_instance.add_key_value_pair_to_db(self.message_id, connection)
                 self.logger.info("Save to database complete.")
                 self.logger.info("Message ID:" + str(self.message_id))
                 self.message_id += 1
                 self.logger.info("Sending connection info to OXP.")
-                # Uncomment lines below to send connection info to OXP
-                try:
-                    r = requests.post(str(OXP_CONNECTION_URL), json=msg_json)
-                    self.logger.info(f"Status from OXP: {r}")
-                except Exception as e:
-                    self.logger.error(f"Error on POST to {OXP_CONNECTION_URL}: {e}")
-                    self.logger.info(
-                        "Check your configuration and make sure OXP service is running."
-                    )
+                # send connection info to OXP
+                if msg_json["operation"] == "post":
```

Comment:
`msg_json.get("operation")` would be better here -- we'll crash if the message has no `operation` field.